File: HW1/src/python/3.py
import pandas as pd
import numpy as np

test_url = "http://s3.amazonaws.com/assets.datacamp.com/course/Kaggle/test.csv"
test = pd.read_csv(test_url)

# Initialize a Survived column to 0
test["Survived"] = 0

# Set Survived to 1 if Sex equals "female" and print the `Survived` column from `test_one`
test["Survived"][test["Sex"] == "female"] = 1

PassengerId =np.array(test["PassengerId"]).astype(int)
my_solution = pd.DataFrame(test[["Survived"]].as_matrix(), PassengerId, columns = ["Survived"])
my_solution.to_csv("my_solution_two.csv", index_label = ['PassengerId'])

File: HW1/src/other/3.R
# Import the testing set: test
test_url <- "http://s3.amazonaws.com/assets.datacamp.com/course/Kaggle/test.csv"
test <- read.csv(test_url)

PassengerId <- as.vector(test$PassengerId)
test_one <- data.frame(PassengerId)
test_one$Survived <- 0
test_one$Survived[test$Sex == "female"] <- 1 
write.csv(test_one, file = "second_prediction.csv",row.names=FALSE)

